tutorials/OnnxRuntimeServerSSDModel.ipynb

```diff
@@ -6,9 +6,7 @@
    "source": [
     "# Inferencing SSD ONNX model using ONNX Runtime Server\n",
     "\n",
-    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
-    "\n",
-    "Licensed under the MIT License.\n",
+    "SPDX-License-Identifier: Apache-2.0\n",
```

Comment:
since this text is visible in the notebook cell, it should say something like Licensed under Apache 2.0 instead of SPDX, right? what's the best practice for notebooks?

Reply:
good point. I think it makes sense to use a sentence rather than an identifier here.